composer.json

```diff
@@ -15,6 +15,7 @@
         "drupal/core-composer-scaffold": "^11@alpha",
         "drupal/core-recommended": "^11@alpha",
         "drush/drush": "^13@beta",
+        "chi-teck/drupal-code-generator": "4.x-dev#e61b917 as 3.5.0",
```

Comment:
This package was updated to Symfony 7, but there is no release.

Considering that the whole `11.x` branch is "early preview", I think this should be okay to have like this.

Once Drupal 11 is in `beta` and all these packages have their releases, we can clean it up.